src/netconf_subscribed_notifications.c

```diff
@@ -627,6 +683,11 @@ csn_config_sub_receivers_prepare(const struct lyd_node *node_receiver,
         goto cleanup;
     }
 
+    rc = csn_send_notif_one(&receiver, nc_sub_id, "started");
+    if (rc) {
+        WRN("Could not send subscription started\n");
+    }
```

Comment:
`rc` is unused, you can put the function call in the `if` directly. Also, the macros always add `\n` at the end so please replace it with `.` instead. And maybe the message itself could be a bit more verbose such as `Could not send notification <subscription-started>.`. Similar problem repeated.